javascript/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@dmsnell/diff-match-patch",
+  "version": "1.0.0",
+  "description": "Active fork of Google's diff-match-patch",
```

Comment:
We shouldn't need to refer to the source of the fork here. Effectively this is a new project now. 🤷‍♂️ 

maybe something like "Efficient and human-readable text diffs with helpers."?